kasa/experimental/smartcameraprotocol.py

```diff
@@ -63,37 +75,63 @@ async def close(self) -> None:
         """Close the underlying transport."""
         await self._transport.close()
 
+    @staticmethod
+    def _get_smart_camera_single_request(
+        request: dict[str, dict[str, Any]],
+    ) -> tuple[str, str, str, dict]:
+        method = next(iter(request))
+        if method == "multipleRequest":
+            method_type = "multi"
+            params = request["multipleRequest"]
+            req = {"method": "multipleRequest", "params": params}
+            return "multi", "multipleRequest", "", req
```

Comment:
Would it make sense to use the `smartcall` dataclass here? That, or a named tuple to make this easier to comprehend.

Reply:
Done